docs/self-managed/about-self-managed.md

```diff
@@ -1,35 +1,41 @@
 
-However, Optimize documentation is available for both Camunda 8 and Camunda 7, including deployment instructions available [here]($optimize$/self-managed/optimize-deployment/install-and-start). Look for `Camunda 7` badges to help you understand what content is available for what product.
-
 :::
 
-As an alternative to using Camunda 8 through SaaS, you can host it yourself; we call this setup Camunda 8 Self-Managed.
+Camunda 8 Self-Managed is a self-hosted alternative to using Camunda 8 SaaS. Building process automation solutions with Camunda 8 works similarly regardless of hosting and deployment.
 
-Building process automation solutions with Camunda 8 Self-Managed is similar to working with Camunda 8 SaaS. For more information on Camunda 8 SaaS, visit [What is Camunda 8?](../components/concepts/what-is-camunda-8.md) If you are new to Camunda 8, we recommend you start your journey with [Camunda 8 SaaS-based guides](../../guides/).
+For more information on Camunda 8 SaaS, visit [What is Camunda 8?](../components/concepts/what-is-camunda-8.md) If you are new to Camunda 8, we recommend you start your journey with [Camunda 8 SaaS-based guides](../../guides/).
 
 The content in this section of the documentation includes:
 
 - Everything you need to download, configure, and work with each component of Camunda 8 Self-Managed.
 - Features specific to Camunda 8 Self-Managed.
 
-The following components are available for Camunda 8 Self-Managed:
+To get started with your Self-Managed installation, see our [installation overview](./setup/overview.md).
+
+## Components
+
+<Components/>
+
+## Architecture
+
+Camunda 8 Self-Managed consists of multiple web applications and gRPC services. The following example architecture diagram uses two [Ingress objects](./setup/guides/ingress-setup.md#combined-ingress-setup) to access these services externally: an Ingress with HTTP(S) protocol for all web applications using a single domain, and another Ingress with gRPC protocol for the Zeebe workflow engine.
+
+![Camunda 8 Self-Managed Architecture Diagram - Combined Ingress](./assets/camunda-platform-8-self-managed-architecture-diagram-combined-ingress.png)
 
-- Zeebe
-- Zeebe Gateway
-- Operate
-- Tasklist
-- Connectors
-- Optimize
-- Identity (not available in Camunda 8 SaaS)
-- Console [<span class="badge badge--enterprise-only">Enterprise only</span>](../../reference/licenses/#console)
-- Web Modeler
+In this configuration, Camunda 8 Self-Managed can be accessed as follows:
 
-Camunda 8 Self-Managed users may also use [Desktop Modeler](../../components/modeler/desktop-modeler/install-the-modeler) as an addition to these components. Desktop Modeler can be used by process developers to build BPMN diagrams, DMN diagrams, or [Camunda Forms](../guides/utilizing-forms.md) for automation.
+- Web applications: `https://camunda.example.com/[identity|operate|optimize|tasklist|modeler]`
```

Comment:
We refer to these as Components. Web Applications is not a topology term that we use.  It also does not represent them fully

Reply:
I think the usage here shouldn't capture each component fully, only represent the web interface, and should explicitly separate them out from Zeebe in terms of access.

There's a number of places where we use web app/application like this, so if it shouldn't be used at all, let me know if I should create a separate ticket to update it elsewhere.

Reply:
I could not craft a good definition of when to use components vs. web applications off the top of my head. If I had to reason out loud, it would be the following: Modeler is the component, but Web Modeler is a web application. I don't think Identity is a web application, but I don't know how I would verify that.

For internal resources, I would recommend this [PMM-maintained page](https://confluence.camunda.com/display/HAN/Internal+Guidance+on+Product+Name%2C+Version%2C+and+Packaging). You can also follow up with @karl-heinsen since I hit a dead end [here](https://camunda.slack.com/archives/C0589TC88UU/p1698325079905339).

Reply:
Would "web components" be preferable? The entirety of this section is focused on separating out the different points of access, so "Components" isn't inclusive enough as Zeebe isn't included, and we're also not referring to the entirety of the component - only the method of access. Looking for any term that is inclusive of all components with a served log-in page, and excludes those without.

```suggestion
In this configuration, Camunda 8 Self-Managed can be accessed as follows:
- Web components: `https://camunda.example.com/[identity|operate|optimize|tasklist|modeler]`
  - Web Modeler also exposes a WebSocket endpoint on `https://camunda.example.com/modeler-ws`. This is only used by the application itself and not supposed to be accessed by users directly.
- Keycloak authentication: `https://camunda.example.com/auth`
- Zeebe gateway: `grpc://zeebe.camunda.example.com`
```

Reply:
I would back either "web applications" or "web components" to get this out the door.

Reply:
Web Modeller has three parts - NodeJS server that exposes web interface, REST API server and Socket server. Identity has also web interface.  'Web Component' is to restrictive and diminishes the value of our product. We typically refer to zeebe, optimize, operate as Component or C8.
Can we use 'Component' going forward?